examples/OneDShallowWaterGeostrophicAdjustment.jl

```diff
@@ -30,12 +30,8 @@ function calcN!(N, sol, t, s, v, p, g)
   mul!(v.uh, g.rfftplan, deepcopy(v.u))
   mul!(v.vh, g.rfftplan, deepcopy(v.v))
   mul!(v.ηh, g.rfftplan, deepcopy(v.η))
-  ldiv!(v.ux, g.rfftplan, -im*g.kr.*v.uh)
-  ldiv!(v.vx, g.rfftplan, -im*g.kr.*v.vh)
-  mul!(v.uuxh, g.rfftplan, v.u.*v.ux)
-  mul!(v.uvxh, g.rfftplan, v.u.*v.vx)
-  rhsu = p.f*v.vh + p.g*p.h*im*g.kr.*v.ηh - v.uuxh
-  rhsv = - p.f*v.uh - v.uvxh
+  rhsu = p.f*v.vh + p.g*p.h*im*g.kr.*v.ηh
+  rhsv = - p.f*v.uh
   rhsη = im*g.kr.*v.uh
```

Comment:
If eq. (3) is correct, then shouldn't come with a minus sign?
```julia
rhsη = - im*g.kr.*v.uh
```

Reply:
But shouldn't it be positive, i.e., `rhsη = im*g.kr.*v.uh`, because it came from the LHS?

Reply:
Perhaps I'm too jet lag, but shouldn't it be
LHS = + ∂u/∂x => RHS = - ∂u/∂x = `- im * g.kr * v.uh` ?

Reply:
But isn't there the minus sign that comes with `im .* g.kr`? i.e., RHS = FFT(-∂u/∂x) = `- ( - im * g.kr .* v.uh)`?

Reply:
What "minus sign that comes with im .* g.kr`"?

`FFT(∂u/∂x) = im*g.kr*v.uh`

Reply:
I've been travelling for long and I might be missing something obvious here..

Reply:
Oh sorry, my mistake. Then equation 1 also has a sign mistake:
`rhsu = p.f*v.vh + p.g*p.h*im*g.kr.*v.ηh` should be `rhsu = p.f*v.vh - p.g*p.h*im*g.kr.*v.ηh`

Reply:
ok, when you fix it I'll merge